main_test.go

```diff
@@ -185,17 +185,24 @@ var _ = Describe("github-review-helper", func() {
 					})
 
 					Context("with GitHub request succeeding", func() {
-						var repo *MockRepo
+						var (
+							repo *MockRepo
+
```

Comment:
Nope. Not necessary. Just thought it's nice to separate these 2 types of things from each other.